lib/internal/validators.js

```diff
@@ -403,9 +403,13 @@ function validateUnion(value, name, union) {
   }
 }
 
-function validateLinkHeaderValue(value, name) {
-  const linkValueRegExp = /^(?:<[^>]*>;)\s*(?:rel=(")?[^;"]*\1;?)\s*(?:(?:as|anchor|title)=(")?[^;"]*\2)?$/;
+const linkValueRegExp = /^(?:<[^>]*>;)\s*(?:rel=(")?[^;"]*\1;?)\s*(?:(?:as|anchor|title)=(")?[^;"]*\2)?$/;
```

Comment:
That is not generic imho.

Reply:
This code refers to Link header having a valid format. Well I was thinking the same thing, until I read the spec. You can find the reference in here: https://datatracker.ietf.org/doc/html/rfc8297#section-2

Reply:
I cant see any abnf format of the Link. Only some examples. There is no anchor or title attribute in the spec

Reply:
Spec is refering with those but should be extended to all link tag attributes for example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link

`crossorigin` is a requirement to me.

Reply:
You're right! Thanks for the feedback. Well, I found the correct HTML standard. It seems all of [these attributes](https://html.spec.whatwg.org/multipage/semantics.html#the-link-element) should be supported. @mcollina can you verify?

Reply:
not sure, what's the question? The issue in question is about the API, not the regexp. If the regexp is not sufficient, maybe open a new issue?

Reply:
Well I'll open a new pull request for the regexp change.

Reply:
I opened another pull-request to discuss which attributes are important to support & add missing ones: https://github.com/nodejs/node/pull/44874